api/v1alpha1/dnshealthcheckprobe_types.go

```diff
@@ -31,9 +31,9 @@ type DNSHealthCheckProbeSpec struct {
 	// Hostname is the value sent in the host header, to route the request to the correct service
 	// +kubebuilder:validation:Pattern=`^[a-z][a-z0-9\-]+\.([a-z][a-z0-9\-]+\.)*[a-z][a-z0-9\-]+$`
 	Hostname string `json:"hostname,omitempty"`
```

Comment:
should these be required?

Reply:
They are. Unless you use `+optional` it thinks it is required.  [Provider ref ](https://github.com/Kuadrant/dns-operator/blob/main/api/v1alpha1/dnsrecord_types.go#L95-L96), for example.